website/cue/reference/components/transforms/filter.cue

```diff
@@ -27,15 +27,17 @@ components: transforms: filter: {
 	configuration: {
 		condition: {
 			description: """
-				The condition to be matched against every input event. Only messages that pass the condition will
-				be forwarded.
+				The condition to be matched against every input event. Only messages that pass the condition are
+				forwarded; messages that don't pass are dropped.
```

Comment:
Maybe we could just add a note in prose here in the description that, in addition to the `type`/`source` map below, there is a short-hand of just passing a VRL string condition? I realize that sort of configuration field type union is difficult to model.